CHANGELOG.md

```
@@ -1,5 +1,30 @@
 # Changelog
 
+## v0.4.6 (unreleased)
+
+### Accessibility
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)